### ISSUE TITLE
Devices in zone being set despite zone not being selected

### ISSUE THREAD

**Issue by ks2018ks** [mskg/homey-heating#168]
Hi,

I have several zones with HeatIt thermostats for electric under floor heating as well as Popp trvs for radiators. Heating schedules are set with only the HeatIt thermostats selected as devices and no zones selected, but the radiator trvs are adopting the HeatIt set temperature controlled by heating schedules. Is this a bug or am I doing something wrong?

**Comment by mskg** [comment]
Are you able to post come screenshots of your configuration? Than it's easier to talk about the same thing.

**Comment by ks2018ks** [comment]
Is this what you mean?
![Heating Scheduler problem](https://user-images.githubusercontent.com/37977911/142831112-ef84010c-dea7-4d9d-ac48-86e36b824afd.png)

**Comment by mskg** [comment]
Super! 

First question: I've seen from the screenshot that you only ticked devices but not a zone. Is that correct?

For your understanding: If you tick a zone, it automatically includes all the temperature devices that are in that zone. This way you don't need to mark individual thermostats if you already organized them in Homey and those share the same temperature.

**Comment by mskg** [comment]
Second: Be sure to set the following settings on. This way you see when the App changes one of the thermostats.

In general, the App only changes the thermostats at the time you requested it in the configuration. It doesn't monitor them or forces the value if you change them manually to a different setting.

![image](https://user-images.githubusercontent.com/46903097/142834850-9a54f6c2-38bb-4ad6-aaee-36a67b948b58.png)

**Comment by ks2018ks** [comment]
Yes, that's how I thought it was supposed to work.  The reason for not selecting the zone, is because the HeatIt thermostat is working on the floor temperature whereas the Popp is for the radiator, so they're not both using a single 'zone' temperature

**Comment by ks2018ks** [comment]
> Second: Be sure to set the following settings on. This way you see when the App changes one of the thermostats.
> 
> In general, the App only changes the thermostats at the time you requested it in the configuration. It doesn't monitor them or forces the value if you change them manually to a different setting.
> 
Those notifications are indeed on

**Comment by mskg** [comment]
> Yes, that's how I thought it was supposed to work.  The reason for not selecting the zone, is because the HeatIt thermostat is working on the floor temperature whereas the Popp is for the radiator, so they're not both using a single 'zone' temperature   

You can deduct from the screenshot. The tabs state (0) the number of ticks in the title.

Can you ensure that you definitely haven't ticked the same device twice? If there is a conflict, the system chosen the higher temperature.

**Comment by ks2018ks** [comment]
I'm not sure what you mean.  My understanding is that:

if you tick a zone, Heating Scheduler tries to set set points for all devices in the zone, irrespective of whether the devices is also selected.  Is that right?

That's why I didn't select any zone, just devices for a given schedule plan.

The zone "Main Bed Bathroom" has two devices.  Each of these devices has a separate Schedule plan (see above).  The device is not selected in any other Schedule and the zone does not have any other schedule plans.

For some reason, the set point of the Popp Radiator TRV appears to be being set according to the Schedule for the other device in the same zone - as if the zone had been ticked.

I have 3 bathrooms set-up the same way.  The same behaviour seems to be true for all 3.  Is there something I am setting incorrectly?
